### PR TITLE
feat(FEC-9972): floating player

### PR DESCRIPTION
**Pull request by RoyBregman**
added new plugin configuration to support new floating player feature under "Look and feel" category

Solves FEC-9972

### CODE REVIEW

File: app/js/services/v3Properties.json [hunk 1]
```diff
@@ -277,6 +277,104 @@
...
+      },
+      "visibility": {
+        "type": "featuremenu",
+        "description": "Viewability plugin handles player behaviour when in / out of view",
+        "model": "config.plugins.visibility",
+        "componentName": "playkit-visibility",
+        "sections": {
+          "tabset": [{
+            "label": "Floating",
+            "model": "floating",
```
Comment (yairans) [inline app/js/services/v3Properties.json:289]: check if needed

File: app/js/services/v3Properties.json [hunk 2]
```diff
@@ -277,6 +277,104 @@
...
+            "key": "floating",
+            "componentName": "playkit-visibility",
+            "active": true,
+            "children": [
```
Comment (yairans) [inline app/js/services/v3Properties.json:293]: check

File: app/js/services/v3Properties.json [hunk 3]
```diff
@@ -277,6 +277,104 @@
...
+            "helpnote": "floating player position"
+          },
+          {
+            "label": "Width",
+            "type": "number",
+            "section": "floating",
+            "model": "config.plugins.visibility.floating.width",
+            "helpnote": "floating player width in pixels",
+            "min": 0,
+            "initvalue": 400
```
Comment (yairans) [inline app/js/services/v3Properties.json:340]: "filter": "numberOnly"
also for others

File: app/js/services/v3Properties.json (hunk 1 above)
Comment (RoyBregman) [reply app/js/services/v3Properties.json:289]: not needed, removed

File: app/js/services/v3Properties.json (hunk 2 above)
Comment (RoyBregman) [reply app/js/services/v3Properties.json:293]: needed

**Pull request merged**